api/src/main/java/uk/gov/dhsc/htbhf/claimant/entity/Claimant.java

```diff
@@ -52,6 +55,9 @@
     @Column(name = "date_of_birth")
     private LocalDate dateOfBirth;
 
+    @OneToOne(mappedBy = "claimant", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
Do we want this to be lazy loaded, wouldn't we normally want the address available straight away too? I'm not sure, its just a question on whether we should load it at the same time as the Claimant we'd normally need it.